nailgun/entities.py

```diff
@@ -7910,7 +7910,7 @@ def __init__(self, server_config=None, **kwargs):
             'hypervisor_password': entity_fields.StringField(),
             'hypervisor_server': entity_fields.StringField(required=True),
             'hypervisor_type': entity_fields.StringField(
-                choices=['esx', 'rhevm', 'hyperv', 'xen', 'libvirt', 'kubevirt'],
+                choices=['esx', 'rhevm', 'hyperv', 'xen', 'libvirt', 'kubevirt', 'ahv'],
```

Comment:
Question, are `rhevm` and `xen` supported hypervisor_types, can't see these choices in apidoc?
```
foreman_virt_who_configure_config[hypervisor_type]                 required | Hypervisor type          Validations:                       Must be one of: esx, hyperv, libvirt, kubevirt, ahv.
-- | --

```

Reply:
Yes, from and apidoc, virtwhoconfig api only support esx, hyperv, libvirt, kubevirt and ahv, I agree with you we can remove these two types.